src/components/balances/TokenListSelect/index.tsx

```diff
@@ -15,6 +17,11 @@ const TokenListLabel = {
   [TOKEN_LISTS.ALL]: 'All tokens',
 }
 
+const enum TokenListActions {
+  OPEN = 'Open token list',
+  CLOSE = 'Close token list',
```

Comment:
I would suggest making two separate objects in the analytics/events enum that would differ only by their label.
E.g.
```
OPEN_TOKEN_LIST_MENU: {
  ...
},
CLOSE_TOKEN_LIST_MENU: {
  ...
}
```

Reply:
Nice suggestion! I've addressed it.